File: calculadora.py
#a basic calculator using variables, if and elses:

number_a = int(input('Digit a number: '))
math_operator = int(input('Choose an math operator: 1)Plus  2)Minus  3)Division  4)Multiplication  5)Exponentiation :  '))
number_b = int(input('Digit another number: '))

if math_operator == 1:
	answer = number_a + number_b
elif math_operator == 2:
	answer = number_a - number_b
elif math_operator == 3:
	answer = number_a / number_b
elif math_operator == 4:
	answer = number_a * number_b
elif math_operator == 5:
	answer = number_a ** number_b

print(answer)

additional = int(input("Want add more number in the operator?  yes = 1  no = 2"))

while additional == 1:
	math_operator2 = int(input('Choose an math operator: 1)Plus  2)Minus  3)Division  4)Multiplication  5)Exponentiation :  '))
	number_c = int(input('Digit another number: '))
	if math_operator2 == 1:
		answer = answer + number_c
		print(answer)
		additional = int(input("Want add more number in the operator?  yes = 1  no = 2"))
	elif math_operator2 == 2:
		answer = answer - number_c
		print(answer)
		additional = int(input("Want add more number in the operator?  yes = 1  no = 2"))
	elif math_operator2 == 3:
		answer = answer / number_c
		print(answer)
		additional = int(input("Want add more number in the operator?  yes = 1  no = 2"))
	elif math_operator2 == 4:
		answer = answer * number_c
		print(answer)
		additional = int(input("Want add more number in the operator?  yes = 1  no = 2"))
	elif math_operator2 == 5:
		answer = answer ** number_c
		print(answer)
		additional = int(input("Want add more number in the operator?  yes = 1  no = 2"))
	else:
		print('An error ocurred or Wrong input.')

if additional == 2:
	print('your result is ' + answer)

File: JavaScript/calculadora/calculadora.js
// a calculator with .
let numberA, numberB, numberC, operator, aditional, result;
function calculate(num1, operator, num2) {
    let result;
    switch(operator) {
        case 1:
            result = num1 + num2;
            break;
        case 2:
            result = num1 - num2;
            break;
        case 3:
            result = num1 * num2;
            break;
        case 4:
            result = num1 / num2;
            break;
        case 5:
            result = num1 ** num2;
            break;
    } return result;
}
do {
    numberA = Number(prompt("Enter the first number."));
    operator = Number(prompt(`Enter the operator by number:\n1-Plus\n2-Minus\n3-Multiplication\n4-Division\n5-Exponentiation`));
    numberB = Number(prompt("Enter the second number."));
    if (operator < 1 || operator > 5) {
        alert("operator: wrong input.");
    }
} while (operator < 1 || operator > 5);

result = calculate(numberA, operator, numberB);
alert(result);

do {
    aditional = prompt("Add another number? (y/n)").toLowerCase();
    if (aditional !== "y" && aditional !== "n") {
        alert("aditional: wrong input.");
    }
} while (aditional !== "y" && aditional !== "n");
    while (aditional === "y") {
    do {
        operator = Number(prompt(`Enter the operator by number:\n1-Plus\n2-Minus\n3-Multiplication\n4-Division\n5-Exponentiation`));
    numberC = Number(prompt("Enter the third number."));
    if (operator < 1 || operator > 5) {
        alert("operator: wrong input.");
    }
    } while (operator < 1 || operator > 5);

result = calculate(result, operator, numberC);
alert(result);
do {
    aditional = prompt("Add another number? (y/n)").toLowerCase();
    if (aditional !== "y" && aditional !== "n") {
        alert("aditional: wrong input.");
    }
} while (aditional !== "y" && aditional !== "n");
}
if (aditional === "n") {
    alert(`The result is ${result}`);
}